Sources/Shared/SharedStrings.swift

```diff
@@ -43,7 +39,7 @@ extension Strings {
   public static let backTitle = NSLocalizedString("BackTitle", bundle: .strings, value: "Back", comment: "Label to display in the Discoverability overlay for keyboard shortcuts")
   public static let forwardTitle = NSLocalizedString("ForwardTitle", bundle: .strings, value: "Forward", comment: "Label to display in the Discoverability overlay for keyboard shortcuts")
 
-  public static let findTitle = NSLocalizedString("FindTitle", bundle: .strings, value: "Find", comment: "Label to display in the Discoverability overlay for keyboard shortcuts")
+//  public static let findTitle = NSLocalizedString("FindTitle", bundle: .strings, value: "Find", comment: "Label to display in the Discoverability overlay for keyboard shortcuts")
```

Comment:
Assume this one was meant to be deleted too?

Reply:
Fixed at https://github.com/brave/brave-ios/pull/6110/commits/e588f9f38d2e9a80b7c79bc9e02267c7562c7983